includes/admin/class-wc-stripe-link-controller.php

```
@@ -0,0 +1,77 @@
+			plugins_url( 'build/link-settings.js', WC_STRIPE_MAIN_FILE ),
+			$asset_metadata['dependencies'],
+			$asset_metadata['version'],
+			true
+		);
+		wp_set_script_translations(
+			'wc-stripe-link-settings',
+			'woocommerce-gateway-stripe'
+		);
+		wp_enqueue_script( 'wc-stripe-link-settings' );
+
+		$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'admin_scripts'. (undefined)

(StaticAccess)